cmd/fts/cmd/resume.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	resumeCmd.Flags().BoolVarP(&dryRun, "dry-run", "d", false, "Run through steps without pushing data to Powergate API.")
+	resumeCmd.Flags().BoolVarP(&mainnet, "mainnet", "n", false, "Sets staging limits based on localnet or mainnet.")
+	resumeCmd.Flags().BoolVarP(&retryErrors, "retry", "e", false, "Retry tasks with error status")
+	resumeCmd.Flags().StringVar(&ipfsrevproxy, "ipfsrevproxy", "127.0.0.1:6002", "Powergate IPFS reverse proxy multiaddr")
+	resumeCmd.Flags().StringVarP(&resultsOut, "results", "r", "results.json", "The of ongoing tasks to resume.")
+
+	// Not included in the public commands
+	resumeCmd.Flags().Int64Var(&concurrent, "concurrent", 10000, "Max concurrent tasks being processed")
+	resumeCmd.Flags().Int64Var(&maxStagedBytes, "max-staged-bytes", 0, "Maximum bytes of all tasks queued on staging")
+	resumeCmd.Flags().Int64Var(&maxDealBytes, "max-deal-bytes", 0, "Maximum bytes of a single deal")
+	resumeCmd.Flags().Int64Var(&minDealBytes, "min-deal-bytes", 0, "Minimum bytes of a single deal")
+	resumeCmd.Flags().MarkHidden("concurrent")
+	resumeCmd.Flags().MarkHidden("max-staged-bytes")
+	resumeCmd.Flags().MarkHidden("max-deal-bytes")
+	resumeCmd.Flags().MarkHidden("min-deal-bytes")
+
+	rootCmd.AddCommand(resumeCmd)
+}
+
+var resumeCmd = &cobra.Command{
+	Use:   "resume",
```

Comment:
realized it was more intuitive that running the same tasks again to resume. so generally just, `fts resume` in the same directory should pick up where you left off.